kf_model_fhir/ingest_plugin/target_api_builders/sequencing_observation.py

```diff
@@ -0,0 +1,142 @@
+"""
+Builds FHIR Observation resources (https://www.hl7.org/fhir/observation.html)
```

Comment:
Maybe you already have this in another place in the model, but if not, can you briefly describe what the sequencing observation represents/what it is?